manifests/uno.ui-preview.manifest.json

```diff
@@ -120,28 +117,12 @@
               "version": "$(MACCATALYST_SDK_VERSION)",
               "supportedPlatforms": [ "Windows", "OSX" ]
             },
-            {
-              "workloadId": "tvos",
-              "workloadManifestId": "microsoft.net.sdk.tvos",
-              "packageId": "Microsoft.NET.Sdk.tvOS.Manifest-6.0.400",
-              "version": "$(TVOS_SDK_VERSION)",
-              "supportedPlatforms": [ "Windows", "OSX" ]
-            },
             {
               "workloadId": "macos",
               "workloadManifestId": "microsoft.net.sdk.macos",
               "packageId": "Microsoft.NET.Sdk.macOS.Manifest-6.0.400",
               "version": "$(MACOS_SDK_VERSION)",
               "supportedPlatforms": [ "Windows", "OSX" ]
-            },
-            {
-              // This workload is required to avoid ios/android workloads to rollback to
-              // maui-pre versions. (Required at RC2 timeframe)
```

Comment:
Just double checking to make sure this was removed intentionally, feel free to resolve this comment!

Reply:
Yes :)